patches/minecraft/net/minecraft/entity/EntityList.java.patch

```diff
@@ -63,7 +63,27 @@
      @Nullable
      public static Entity func_191304_a(@Nullable Class <? extends Entity > p_191304_0_, World p_191304_1_)
      {
-@@ -168,13 +180,15 @@
+@@ -152,6 +164,12 @@
+         }
+         else
+         {
++            net.minecraftforge.fml.common.registry.EntityEntry entry = net.minecraftforge.fml.common.registry.EntityRegistry.getEntry(p_191304_0_);
++            return entry == null ? null : entry.newInstance(p_191304_1_);
++        }
++        /*
++        else
++        {
```

Comment:
Uncomment this and remove the extra `else {`. Instead, change L71 to `if(true) return entry == null ? null : entry.newInstance(p_191304_1_);` This stops the compiler from complaining about the dead code, and keeps the patch small and less likely to break.